private/buf/cmd/buf/command/config/configinit/configinit.go

```diff
@@ -90,6 +90,11 @@ func (f *flags) Bind(flagSet *pflag.FlagSet) {
 		".",
 		`The directory to write the configuration file to.`,
 	)
+	// TODO(alex): Now that 'buf config init' is built to (eventually) be general to all buf
+	// configuration files, should we still support the module name and dependencies flags?
+	//
+	// We might be able to avoid these compatibility concerns if we pitch this command as one
+	// that is subject to change over time (as discussed).
```

Comment:
Let's discuss whether or not we should remove the `buf config init --name` flag. The only reason I call it out is if/when we extend `buf config init` to other configuration files, then the semantics of `--name` might get confusing (i.e. if multiple configurations had their own concept of `name`).

Reply:
Discussed offline - we'll drop these flags in a follow-up. Some tests will need to be adjusted.